optimade/models/entries.py

```diff
@@ -58,8 +58,8 @@ class EntryResourceAttributes(Attributes):
 - **Type**: string.
 - **Requirements/Conventions**:
 
-  - **Response**: OPTIONAL in the response.
-  - **Query**: If present, MUST be a queryable property with support for all mandatory filter operators.
+  - **Support**: OPTIONAL support in implementations, i.e., MAY be :val:`null`.
```

Comment:
```suggestion
  - **Support**: OPTIONAL, i.e., MAY be :val:`null`.
```

Reply:
All this is directly from the specification. It is simply an update. I think I need to do the same for all field descriptions since these were changed with Materials-Consortia/OPTiMaDe#210